cyclic_boosting/base.py

```diff
@@ -338,11 +339,21 @@ def _check_weights(self):
         if self.weights is None:
             raise RuntimeError("The weights have to be initialized.")
 
-    def _init_features(self):
+    def _init_features(self, X=None):
```

Comment:
You better do this here: https://github.com/Blue-Yonder-OSS/cyclic-boosting/blob/490df233c6acfe7b870228184cf4cf5b887f30f3/cyclic_boosting/base.py#L602